### PR TITLE
Update ReadMe, Site docs and fix broken links

### PR DESCRIPTION
**Pull request by ajantha-bhat**

### CODE REVIEW

File: site/docs/features/management.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ The above snippet assumes a `TABLE_IDENTIFIER` which points to the unreferenced
 spark session and a nessie owned `Catalog`. The `result` object above returns the number of files the action tried to delete and the number that failed.
 
 !!! note
-    You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-demo-nba.ipynb)
+    You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/notebooks/nessie-iceberg-demo-nba.ipynb)
```
Comment (snazy) [inline site/docs/features/management.md:102]: This should point to the Binder notebook, we no longer support Google Colab.

Comment (ajantha-bhat) [reply site/docs/features/management.md:102]: yeah, below this line it has binder link as well. I shall remove colab link then.

File: site/docs/features/management.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ Key configuration parameters:
 ## Compaction
 
 !!! info
-This service is currently in progress and is not yet included in a released version of Nessie.
+    This service is currently in progress and is not yet included in a released version of Nessie.
```
[the remark this answers is not in the document]
Comment (ajantha-bhat) [reply site/docs/features/management.md:166]: yes. I will look up all the doc once and handle it.

File: site/docs/features/management.md (hunk 1 above)
Comment (ajantha-bhat) [reply site/docs/features/management.md:102]: @snazy :  what about links to this (nessie-iceberg-spark-setup.ipynb) notebook ? I saw that recently during clean up this document is deleted. So, I will remove entire text talking about this document ?

File: site/docs/features/management.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ajantha-bhat) [reply site/docs/features/management.md:166]: handled. only this place was pending

File: README.md [hunk 3]
```diff
@@ -22,18 +22,22 @@ docker run -p 19120:19120 projectnessie/nessie
 
 A local [Web UI](https://projectnessie.org/tools/ui/) will be available at this point.
 
-Then install the Nessie CLI tool
+Then install the Nessie CLI tool (to learn more about CLI tool and how to use it, check [here](https://projectnessie.org/tools/cli/)
```
Comment (dimas-b) [inline README.md:25]: nit: `here` -> `Project Nessie Documentation` ?

**Review by dimas-b — APPROVED** [review]

File: site/docs/features/management.md (hunk 1 above)
Comment (snazy) [reply site/docs/features/management.md:102]: No, don't delete it. Just replace the link with
```suggestion
    You can follow along an interactive demo in [this Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-demo-nba.ipynb)
```

Comment (ajantha-bhat) [reply site/docs/features/management.md]: ok. I have changed as per your suggestion

File: README.md (hunk 3 above)
Comment (ajantha-bhat) [reply README.md]: ok. changed it to `Nessie CLI Documentation`

**Comment by ajantha-bhat** [comment]
@snazy : I have gone through ReadMe and all the siteDocs. I didn't found any another issues. I think this is all I have for this PR. Please check it.

**Review by omarsmak — APPROVED** [review]

**Review by snazy — APPROVED** [review]

**Review by dimas-b — APPROVED** [review]

**Pull request merged**